velox/dwio/parquet/tests/writer/ParquetWriterTest.cpp

```diff
@@ -748,6 +807,23 @@ TEST_F(ParquetWriterTest, parquetWriteTimestampTimeZoneWithDefault) {
   writer->close();
 };
 
+DEBUG_ONLY_TEST_F(ParquetWriterTest, parquetWriteWithArrowMemoryPool) {
```

Comment:
Perhaps this test does not need to be debug-only.

Reply:
@rui-mo Good catch. Updated.